pom.xml

```diff
@@ -0,0 +1,192 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.kiwiproject</groupId>
+    <artifactId>kiwi-parent</artifactId>
+    <version>0.3.0</version>
+  </parent>
+
+  <artifactId>dropwizard-service-utilities</artifactId>
+  <version>0.1.0-SNAPSHOT</version>
+  <packaging>jar</packaging>
+
+  <name>${project.groupId}:${project.artifactId}</name>
+  <description>
+    Set of utilities to aide in setting up Dropwizard services
```

Comment:
That would be "aid" 😜

Reply:
(an aide is an assistant to an important person)

Reply:
haha yes thank you